CMakeLists.txt

```diff
@@ -166,19 +166,15 @@ add_boost_if_needed()
 
 include(CheckSourceCompilesAndLinks)
 
-if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.14)
-  include(CheckPIESupported)
-  check_pie_supported(OUTPUT_VARIABLE check_pie_output LANGUAGES CXX)
-  if(CMAKE_CXX_LINK_PIE_SUPPORTED)
-    set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-  endif()
+include(CheckPIESupported)
```

Comment:
In 66f1e5b766d071a53f45e1f8436d9c8d8cc717e1:

Not sure what's happening here, but this check doesn't work, and I don't get PIE.
```bash
CMake Warning at CMakeLists.txt:174 (message):
  PIE is not supported at link time: PIE (CXX): Change Dir:
...
  Position independent code disabled.
```

Full output here: https://gist.github.com/fanquake/1c38b5367a8cca924fd732e7b5fe7817.